raiden-dapp/src/components/AddressInput.vue

```diff
@@ -146,13 +162,37 @@ export default class AddressInput extends Mixins(BlockieMixin) {
       this.errorMessages.push(this.$t(
         'address-input.error.no-checksum'
       ) as string);
-    } else if (AddressUtils.checkAddressChecksum(value)) {
-      this.input(value);
     } else if (
       !AddressUtils.isAddressLike(value) &&
       AddressUtils.isDomain(value)
     ) {
       this.resolveEnsAddress(value);
+    } else if (
+      AddressUtils.checkAddressChecksum(value) &&
+      !Object.keys(this.presences).includes(value)
+    ) {
+      this.isAddressAvailable = false;
```

Comment:
@kelsos these seem to be unreachable lines, according to codecov and I'm not sure how to reach them at all. Any suggestions? The main problem I have is that mocking state in VueJS is really a pain